app/assets/stylesheets/modules/admin.scss

```diff
@@ -45,3 +45,24 @@ h3 + .table.analytics {
   margin-top: $padding-large-vertical * 3;
   padding-bottom: $padding-base-vertical;
 }
+
+.tanslation-description-toggle {
```

Comment:
Is this misspelled upstream?

Reply:
Good eye, but yes: https://github.com/projectblacklight/spotlight/search?q=tanslation-description-toggle&unscoped_q=tanslation-description-toggle

I just copied and pasted so didn't notice. Would kind of prefer not fixing spotlight for this right now, but could make a ticket to fix later.